src/database/schema/ChannelListSchema.ts

```diff
@@ -10,6 +10,7 @@ import {BetterSocialChannelType} from '../../../types/database/schema/ChannelLis
 import {CHANNEL_GROUP, PM} from '../../hooks/core/constant';
 import {ChannelData} from '../../../types/repo/AnonymousMessageRepo/AnonymousChannelsData';
 import {ChannelType} from '../../../types/repo/ChannelData';
+import {DEFAULT_PROFILE_PIC_PATH} from '../../utils/constants';
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The import of `DEFAULT_PROFILE_PIC_PATH` is a good addition to provide a fallback for the profile picture in case `data.targetImage` is not available. This improves the robustness of the code.
```diff
+ import {DEFAULT_PROFILE_PIC_PATH} from '../../utils/constants';
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->